utils/validation.js

```diff
@@ -316,6 +319,30 @@ const checkDerivedVariables = async (token, siteCode) => {
         }
     }
 
+    // anyRefusalWithdrawal
+    if(data['451953807']) {
+        if(data['451953807'] === 104430631) {
```

Comment:
The 2 layers of `if` conditions can be combined: `if (data?.['451953807'] === 104430631)`

Reply:
I was gonna say the same thing

Reply:
will be putting this in, thank you both